lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -1038,6 +1042,8 @@ def install_puppet_agent_dev_repo_on( hosts, opts )
               result = on host, "echo #{onhost_copied_file}"
               onhost_copied_file = result.raw_output.chomp
               on host, Command.new("start /w #{onhost_copied_file}", [], { :cmdexe => true })
+            when /^osx$/
+              host.install_package("puppet-agent-*")
```

Comment:
this seems a little risky, if you have more than one dmg on the coordinator (running acceptance on-demand for multiple SHAs), then which dmg will this install?